packages/synchro-charts/src/components/sc-widget-grid/sc-grid-tooltip.tsx

```diff
@@ -64,7 +70,8 @@ export class ScGridTooltip {
                     <div class="awsui-util-label">Latest value:</div>
                     <div>
                       <strong style={{ color }}>
-                        <Value value={this.propertyPoint.y} />
+                        {icon && <sc-chart-icon name={icon} color={color} style={{ marginRight: '3px' }} />}
+                        <Value value={value + unit} />
```

Comment:
`Value` already has a `unit` prop, is there a reason not to use that instead?

Reply:
Done --- Use `unit` instead